src/main/java/seedu/address/model/category/Category.java

```diff
@@ -1,54 +1,54 @@
-package seedu.address.model.tag;
+package seedu.address.model.category;
 
 import static java.util.Objects.requireNonNull;
 import static seedu.address.commons.util.AppUtil.checkArgument;
 
 /**
  * Represents a Tag in the address book.
- * Guarantees: immutable; name is valid as declared in {@link #isValidTagName(String)}
+ * Guarantees: immutable; name is valid as declared in {@link #isValidCategoryName(String)}
  */
-public class Tag {
+public class Category {
 
     public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric";
     public static final String VALIDATION_REGEX = "\\p{Alnum}+";
 
-    public final String tagName;
+    public final String categoryName;
 
     /**
      * Constructs a {@code Tag}.
      *
-     * @param tagName A valid tag name.
+     * @param categoryName A valid category name.
      */
-    public Tag(String tagName) {
-        requireNonNull(tagName);
-        checkArgument(isValidTagName(tagName), MESSAGE_CONSTRAINTS);
-        this.tagName = tagName;
+    public Category(String categoryName) {
+        requireNonNull(categoryName);
+        checkArgument(isValidCategoryName(categoryName), MESSAGE_CONSTRAINTS);
+        this.categoryName = categoryName;
     }
 
     /**
-     * Returns true if a given string is a valid tag name.
+     * Returns true if a given string is a valid category name.
      */
-    public static boolean isValidTagName(String test) {
+    public static boolean isValidCategoryName(String test) {
         return test.matches(VALIDATION_REGEX);
     }
 
     @Override
     public boolean equals(Object other) {
         return other == this // short circuit if same object
-                || (other instanceof Tag // instanceof handles nulls
-                && tagName.equals(((Tag) other).tagName)); // state check
+                || (other instanceof Category // instanceof handles nulls
+                && categoryName.equals(((Category) other).categoryName)); // state check
     }
 
     @Override
     public int hashCode() {
-        return tagName.hashCode();
+        return categoryName.hashCode();
     }
 
     /**
      * Format state as text for viewing.
      */
     public String toString() {
-        return '[' + tagName + ']';
+        return '[' + categoryName + ']';
```

Comment:
I don't think the '[' and ']' are needed anymore since it is no longer a list

Reply:
Actually, I intend to only push out the changes for Category to only catch the 10 standard categories given by the application instead of any String defined by the user after mid v1.3, so they won't really be checked by a regex anymore.

For this round I think we can just focus on making `categoryName` private instead of public.
Currently the tests and everything else access the `categoryName` attribute which was initially already made public. Do you think it is better to make a getter for `categoryName` and keep the `toString()` or just let `toString()` be the only method to return `categoryName`?